### ISSUE TITLE
the ArrowIcon in the Pagination component is disappeared  when using the local Iconfont

### ISSUE THREAD

**Issue by DiroKate** [ant-design/ant-design#6335]
### Version
2.9.1

### Environment
mac OS X 10.12; chrome the latest; dva 1.2.1;

### Reproduction link
[https://github.com/DiroKate/SysuhikerCC](https://github.com/DiroKate/SysuhikerCC)

### Steps to reproduce
1. Edit my `@icon-url` config, point to the local iconfont file.
2. set the `prefix` of the local iconfont: `anticon`, we can use `<Icon type="xxx" />`. P.S. the **left** and **right** icon is set into the iconfont file.
3. using `<Table>` component, but the pagination arrow is disappeared, only a small square left.

### What is expected?
show the pagination prev&&next Arrow.

### What is actually happening?
the pagination arrow is disappeared, only a small square left.

**Comment by benjycui** [comment]
Please consult https://github.com/ant-design/antd-init/blob/master/examples/local-iconfont/README.md

**Comment by afc163** [comment]
Please give me a reproducible way directly, it is hard to follow your steps.

**Comment by DiroKate** [comment]
@benjycui that's exactly i refer to. I could use the local iconfont.
But the Antd component's icon is disappeared. could we take the user-iconfont and Antd-iconfont at the same time?

**Comment by afc163** [comment]
Try make another icon component like antd, don't mix up the two iconfont files.

**Comment by DiroKate** [comment]
thanks, it helps.
i make a new icon component for local iconfont.
